helpdesk_mgmt/views/helpdesk_ticket_templates.xml

```diff
@@ -466,7 +466,10 @@
                             t-att-required="ticket_category_id_required"
                         >
                             <option value="" />
-                            <t t-foreach="categories" t-as="cat">
+                            <t
+                                t-foreach="categories.filtered('show_in_portal')"
+                                t-as="cat"
+                            >
```

Comment:
IMO this filter should happen in the controller: https://github.com/OCA/helpdesk/blob/a9f1089b3564c9c55757dce9fc0ed1181249727f/helpdesk_mgmt/controllers/main.py#L48-L50